README.md

```diff
@@ -122,6 +122,8 @@ application without specifying the config file location with the `-c` parameter.
 Install `threatbus` and all plugins that you require. Optionally, use a virtual
 environment.
 
+Note that threatbus requires at least Python 3.7+, earlier versions are not supported.
```

Comment:
`s/threatbus/Threat Bus` in text.

Should this also be mirrored to docs.tenzir.com/threatbus?